src/config.ts

```diff
@@ -0,0 +1,38 @@
+import type { MergeableRecord} from './util/objects'
+import {deepMergeObject} from './util/objects'
+import path from 'path'
+import fs from 'fs'
+import {log} from './util/log'
+
+export interface FlowrConfigOptions extends MergeableRecord {
+	ignoreSourceCalls: boolean
+}
+
+export const defaultConfigFile = 'flowr.json'
```

Comment:
i think i weakly recall using this as the example output file for the benchmarking somewhere. Could you please check?

Reply:
Seems like they're all called `benchmark.json` or `benchmark-[something].json`, so doesn't seem to be an issue. Nothing else seems to be called `flowr.json` either.